src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -156,9 +156,7 @@ function MdAutocomplete () {
       // Set our variable for the link function above which runs later
       hasNotFoundTemplate = noItemsTemplate ? true : false;
 
-      if (attr.hasOwnProperty('tabindex')) {
-        element.attr('tabindex', '-1');
-      }
+      if (!tabindex) element.attr('tabindex', '-1');
```

Comment:
This doesn't work because `0` is a valid tabindex that you don't want to overwrite.